src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ModuleBuilderImpl.cs

```diff
@@ -883,19 +883,19 @@ private static int GetTokenForHandle(EntityHandle handle)
 
         private EntityHandle GetHandleForMember(MemberInfo member)
         {
-            if (IsConstructedFromNotBakedTypeBuilder(member.DeclaringType!))
+            if (IsConstructedFromTypeBuilder(member.DeclaringType!))
             {
                 return default;
             }
 
             return GetMemberReferenceHandle(member);
         }
 
-        private static bool IsConstructedFromNotBakedTypeBuilder(Type type) => type.IsConstructedGenericType &&
```

Comment:
These methods initially used for checking if constructed from a not baked `TypeBuilder`, but now the check for not baked part is removed and its used for all cases no matter if the type is baked or not